js/react_native/android/src/androidTest/java/ai/onnxruntime/reactnative/TensorHelperTest.java

```diff
@@ -206,6 +207,7 @@ public void createInputTensor_double() throws Exception {
   }
 
   @Test
+  @Ignore("data type for Slice is not supported in mobile package")
```

Comment:
note: this was working previously because Slice happens to have a single kernel def for multiple types. that kernel's implementation is data type size-based, so it would work since the enabled types would also cover sizeof(bool) and sizeof(double). matching with the kernel def hash happened to work, however, it doesn't work with the updated kernel matching which actually checks the kernel def's types.

if we move to using a full build in the react native package we can enable these tests again.